### PR TITLE
Basic UI

### PR DESCRIPTION
**Pull request by muhammedb8826**
### I have worked on the following: 

- Created an app.rb file that will serve as my console app entry point with methods
- In my main.rb   I define the entry point, this will be a method called main that is invoked at the end of my file. 
This method will do the following:
_Present the user with a list of options to perform.
Lets users choose an option.
If needed, ask for parameters for the option.
Have a way to quit the app._

### CODE REVIEW

File: rental.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    book.rentals << self
+    @person = person
+    person.rentals << self
+  end
+end
```
Comment (mahbubul14) [inline rental.rb]: - [ ] When I try to create a rental, your console app is not working after entering the date. It shows the following error. Kindly fix the issue. :+1: 

terminal error

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @muhammedb8826,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] In your `README` file, Kindly update the getting started section. This should be an example of how you may give instructions on setting up your project locally. Unfortunately, your app is not working in the mentioned way.

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "Fixed bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: rental.rb (hunk 1 above)
Comment (muhammedb8826) [reply rental.rb]: Thank you for the feedback, I'll do that

**Review by amon-cofie — APPROVED** [review]
Hi @muhammedb8826 , 👋 

Your project is complete and very impressive 💯!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

🚀 You fixed the issues pointed out by the previous reviewer
🚀 You used Gitflow properly
🚀 You have no linter errors


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by muhammedb8826** [comment]
Thank you for the approval

**Pull request merged**